aws-test/tests/aws_iam_account_password_policy/test-get-expected.json

```diff
@@ -1,14 +1,14 @@
 [
   {
     "account_id": "{{ output.aws_account.value }}",
-    "allow_users_to_change_password": "{{ output.allow_users_to_change_password.value }}",
-    "expire_passwords": "{{ output.expire_passwords.value }}",
-    "minimum_password_length": "{{ output.minimum_password_length.value }}",
+    "allow_users_to_change_password": {{ output.allow_users_to_change_password.value }},
+    "expire_passwords": {{ output.expire_passwords.value }},
+    "minimum_password_length": {{ output.minimum_password_length.value }},
     "partition": "{{ output.aws_partition.value }}",
     "region": "global",
-    "require_lowercase_characters": "{{ output.require_lowercase_characters.value }}",
-    "require_numbers": "{{ output.require_numbers.value }}",
-    "require_symbols": "{{ output.require_symbols.value }}",
-    "require_uppercase_characters": "{{ output.require_uppercase_characters.value }}"
+    "require_lowercase_characters": {{ output.require_lowercase_characters.value }},
+    "require_numbers": {{ output.require_numbers.value }},
+    "require_symbols": {{ output.require_symbols.value }},
+    "require_uppercase_characters": {{ output.require_uppercase_characters.value }}
```

Comment:
Is it working without `""`?

Reply:
Yes, it is passing if we are removing the `"` otherwise it is failing.
![image](https://user-images.githubusercontent.com/47887552/117427057-3f526980-af42-11eb-9963-f3babaa449c9.png)